CMakeLists.txt

```diff
@@ -156,6 +184,9 @@ if(BUILD_TESTING)
   find_package(benchmark CONFIG REQUIRED)
 endif()
 
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
```

Comment:
Don't include these 2 for non-posix?

Reply:
The ```GNUInstallDirs``` module only defines some directory names to install. And ```CMakePackageConfigHelpers``` is used to generate the standardized cmake config files, they are used by ```find_package(NAME CONFIG)``` of ```cmake```. These two modules are useful on both posix and non-posix platform.